payloads/java/Makefile

```diff
@@ -82,8 +82,8 @@ fromsrc: .check ${JDK_VERSION} jtreg ${KM_BIN} ${KM_LDSO}
 	fi
 	@# -fcommon gcc flag is needed for gcc10, see https://bugs.openjdk.java.net/browse/JDK-8235903
 	cd ${JDK_VERSION} && bash configure \
-		--disable-warnings-as-errors --with-native-debug-symbols=none \
-		--with-extra-cflags='-fcommon'  \
+		--disable-warnings-as-errors --with-native-debug-symbols=internal \
```

Comment:
Yeah, I see what you mean. I think this was just source code entropy in action. Good catch.